players/g1_player.py

```diff
@@ -248,7 +251,11 @@ def __init__(self, skill: int, rng: np.random.Generator, logger: logging.Logger,
     def _max_ddist_ppf(self, conf: float):
         return self.max_ddist.ppf(1.0 - conf)
 
-    def reachable_point(self, current_point: Tuple[float, float], target_point: Tuple[float, float], conf: float) -> bool:
+    def _max_sandtrap_ddist_ppf(self, conf:float):
+        return self.max_sandtrap_ddist.ppf(1.0-conf)
+
+    '''
```

Comment:
Let's delete this? If we need it in the future, we can always find it in the commit history.